api/src/services/streams-service.ts

```diff
@@ -1,9 +1,15 @@
 import { ChannelData, ChannelLog } from '../models/types/channel-data';
 import streams, { Address, Author, Subscriber } from '../streams-lib/wasm-node/iota_streams_wasm';
 import { fromBytes, toBytes } from '../utils/text';
+import * as fetch from 'node-fetch';
 
 streams.set_panic_hook();
 
+global.fetch = fetch as any;
+global.Headers = (fetch as any).Headers;
+global.Request = (fetch as any).Request;
+global.Response = (fetch as any).Response;
```

Comment:
Is it used somewhere?

Reply:
This is a suggestion by streams library. I didn't had problems but jose mentioned this causes problems on some development machines maybe based on node version or operating system idk. That's why I've added it.